airmozilla/starred/static/starred/js/star_event.js

```diff
@@ -70,6 +70,8 @@ var Stars = (function() {
                 // jquery array comparison http://stackoverflow.com/a/7726509/205832
                 if (!($(stars).not(serverStars).length === 0 &&
                       $(serverStars).not(stars).length === 0)) {
+                    stars = stars.concat(serverStars);
```

Comment:
A ha! I get it. The `stars` gets set to `stars = response.ids;` a couple of milliseconds later.